### ISSUE TITLE
Swagger UI renders parameter descriptions and response descriptions in monospace

### ISSUE THREAD

**Issue by DavidBiesack** [swagger-api/swagger-ui#4173]
Swagger UI renders parameter descriptions and response descriptions in monospace

| Q                               | A
| ------------------------------- | -------
| Bug or feature request?         |  bug
| Which Swagger/OpenAPI version?  | 2.0
| Which Swagger-UI version?       | 3.6.1
| How did you install Swagger-UI? |  pull from GigHub and deploy dist  to localhost web server
| Which browser & version?        | Chrome 63.0.3239.132 (Official Build) (64-bit)
| Which operating system?         | OS X Hi Siersa 10.13.1

```yaml
your: "API definition goes here"
```

### Configuration (browser query string, constructor, config.yaml)

```yaml
swagger: '2.0'

info:

  title: Swagger UI Markdown Issue
  description: Is [Markdown](http://commonmark.org/help/) handled in Swagger UI?
  version: '2018-01-29'
basePath: /exampleApi
paths:
  /models/{modelId}:
    parameters:
      - name: modelId
        in: path
        description: Is [Markdown](http://commonmark.org/help/) handled in Swagger UI?
        type: string
        required: true
    get:
      responses:
        '200':
           description: >-
             *OK*

             Is [Markdown](http://commonmark.org/help/) handled in
             Swagger UI?
             
             The response contains a `date` and a unique `id`
           schema: 
             $ref: '#/definitions/model'
             
definitions:

  model:
    properties:
      date:
        description: >-
          The model's date

          Is [Markdown](http://commonmark.org/help/) handled in Swagger UI?
        type: string
        format: date-time
      id:
        description: "The model's unique ID"
        type: string
    example:
      id: 1
      date: 2018-01-22T03:43:57.375Z

```

### Expected Behavior
descriptions and other text should be in normal text font, not monospace.
Text _within_ the descriptions may use backticks to result in monospace,
but

### Current Behavior
 if everything is monospace, such text is not distinguishable.
See screenshot
![screen shot 2018-01-29 at 9 07 28 am](https://user-images.githubusercontent.com/545944/35514379-fd7bc996-04d3-11e8-85a0-0396264c5292.png)

**Comment by DavidBiesack** [comment]
Inspecting in Chrome, I see the text in the table uses (from swagger-ui.css)
```
font-family: Source Code Pro,monospace;
```
(inherited from `table.model`) 
I've no idea where this comes from (i.e. what builds dist/swagger-ui.css)

**Comment by shockey** [comment]
Yeah, this has bothered me for a while now. I'd prefer a sans-serif font on a lighter background (maybe sand or something).

Unfortunately, this is pretty minor in the grand scheme of things, so we're likely to rely on a contribution for this. For anyone interested in tackling it...

The response description is being rendered here: https://github.com/swagger-api/swagger-ui/blob/master/src/core/components/response.jsx#L135
The styles come from here...: https://github.com/swagger-api/swagger-ui/blob/e531c40356a58965a6199924e9af21e4282cc02c/src/style/_layout.scss#L549
... which uses the `text_code` mixin, defined here: https://github.com/swagger-api/swagger-ui/blob/a6bb7f1dd8e381f46d0091a57f00493a9c2fa33e/src/style/_type.scss#L8

**Comment by DavidBiesack** [comment]
thanks for the pointers - no wonder I could not find the source; I looked for *.css and it is in *.scss
I'm not sure if I'm qualified to make a PR; I'm not a front end guy. If I get a chance, I'll look.

**Comment by shockey** [comment]
Operation Markdown blocks look much better now:

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/680248/62594941-47730600-b891-11e9-9818-9a59c621720c.png">

**I'm going to keep this open because Markdown appearance in models is still so-so.**
